src/core/utils/fileutils.cpp

```diff
@@ -845,3 +847,88 @@ bool FileUtils::unzip( const QString &zipFilename, const QString &dir, QStringLi
 
   return true;
 }
+
+bool FileUtils::isDeletable( const QString &filePath )
+{
+  const QFileInfo fileInfo( filePath );
+  if ( !fileInfo.exists() || !fileInfo.isFile() )
+    return false;
```

Comment:
```suggestion
  if ( !fileInfo.exists() || !fileInfo.isFile() )
  {
    return false;
  }
```

I really liked the recent push to kill the braceless blocks. Leave it to @nirvn to have final opinion.